react/src/components/Projects/ProjectListing.tsx

```diff
@@ -16,51 +18,79 @@ export const ProjectListing: React.FC = () => {
     setIsModalOpen(!isModalOpen);
   };
 
-  const { data, isLoading, isError } = useProjectsWithDesignSafeInformation();
+  const { data, isLoading, isError, error } =
+    useProjectsWithDesignSafeInformation();
 
   if (isLoading) {
     return <LoadingSpinner />;
   }
 
   if (isError) {
-    return <h4>Unable to retrieve projects</h4>;
+    return (
```

Comment:
@nathanfranklin If I move the error message out of the table, the Create New Map button never renders. Should I have a header row here like in return statement if there is an error loading maps?

Reply:
Our `error` covers something wrong with the DS project listing and/or Geoapi project. So, I think it's okay to just show an error message and not have buttons or listings (like the Create New Map button).

Usually, if one of those two endpoints returns an error, then it's a sign that they (Geoapi or Design Safe) are down/offline.